src/datasets/providers/index.ts

```diff
@@ -0,0 +1,59 @@
+import type { ComponentType, SVGProps } from 'react';
+
+import { CloudBees } from './cloudbees';
+import { ConfigCat } from './configcat';
+import { DevCycle } from './devcycle';
+import { Flagd } from './flagd';
+import { Flagsmith } from './flagsmith';
+import { Flipt } from './flipt';
+import { Goff } from './goff';
+import { LaunchDarkly } from './launchdarkly';
+import { PostHog } from './posthog';
+import { Split } from './split';
+import { EcosystemElement, Technology } from '..';
+
+export const PROVIDERS: EcosystemElement[] = [
+  CloudBees,
+  ConfigCat,
+  DevCycle,
+  Flagd,
+  Flagsmith,
+  Flipt,
+  Goff,
+  LaunchDarkly,
+  PostHog,
+  Split,
+]
+  .map((i) => {
+    return Object.entries(i.technologies).map(([technology, { vendorOfficial, href }]) => {
+      return {
+        vendor: i.name,
+        title: `${i.name} Provider`,
+        description:
+          typeof i.description === 'undefined'
```

Comment:
Do you actually want to specifically check for undefined? or do a more standard falsey check (`!i.description`)?